### ISSUE TITLE
SelectedEdit : Two-way Binding issue

### ISSUE THREAD

**Issue by brettwinters** [stsrki/Blazorise#251]
Hi @stsrki 

I've discovered some odd binding behavior which seems to change depending on whether a `SelectEdit` is inside a `Field` or not (hard-coding the SelectedValue is not a common use-case, but I discovered this issue while trying to isolate another issue also related to different behavior inside a 'Field' component )

```html
<h5>With Field Wrapper</h5>
<Field>
    <SelectEdit TValue="int"
                SelectedValue="2"
                SelectedValueChanged="@((e) => Console.WriteLine(e))">
        <SelectItem Value="1">one</SelectItem>
        <SelectItem Value="2">two</SelectItem>
        <SelectItem Value="3">three</SelectItem>
    </SelectEdit>
</Field>

<h5>Without Field Wrapper</h5>
<SelectEdit TValue="int"
            SelectedValue="2"
            SelectedValueChanged="@((e) => Console.WriteLine(e))">
    <SelectItem Value="1">one</SelectItem>
    <SelectItem Value="2">two</SelectItem>
    <SelectItem Value="3">three</SelectItem>
</SelectEdit>
```

If the `SelectEdit` is wrapped in a `Field` then I cannot change the selection (although it does print the  selected item correctly in the console). 

Whereas without a field wrapper (or just wrapping in `<div class="form-group">` directly) allows changing the selection.

In both cases, binding the `SelectedValue` and `SelectedValueChanged` to a variable allows the selection to be changed in both cases as expected.

**Comment by stsrki** [comment]
I tried both of you examples and in both of them I cannot change the value. This is in my opinion the right behavior. It's because whats generally is happening is that  `SelectedValueChanged` is of `EventCallback` type. When selected value is changed Blazorise will raise an event and then the Blazor will internally call `StateHasChanged`. This will refresh the `SelectEdit` part and since the initial value is always the same it will reset.

I reported a similar problem few months ago to Blazor team. They also advice to use variable instead of hard-coded values.
